lib/bigcommerce/connection.rb

```diff
@@ -39,19 +43,19 @@ def verify_peer=(verify)
     end
 
     def ssl_ca_file=(path)
-      @configuration.ssl_ca_file = path
+      @configuration[:ssl_ca_file] = path
     end
 
     def ssl_client_key=(path,passphrase=nil)
```

Comment:
@pedelman I like the second option better as going with first option would not be semantically correct, we would be expecting two parameters path and passphrase(optional) and always receive one(the hash) in `path` and then break it down from there into actual params which was our original expectation. 
We could go with the second option and have a new key which calls the old version.  Currently the option is called `ssl_client_key` and it has associated setter available. We could call the new key as`ssl_client_key_details` or may be `ssl_client`. 

Let me know what key name you prefer or if you have any suggestions. :)